External/FEXCore/Source/Interface/Core/JIT/x86_64/JITClass.h

```diff
@@ -290,6 +290,7 @@ class JITCore final : public CPUBackend, public Xbyak::CodeGenerator {
   DEF_OP(FillRegister);
   DEF_OP(LoadFlag);
   DEF_OP(StoreFlag);
+  Xbyak::RegExp GenerateModRM(Xbyak::Reg Base, IR::OrderedNodeWrapper Offset, IR::MemOffsetType OffsetType, uint8_t OffsetScale);
```

Comment:
Move this outside of the definitions of the IR handling ops for cleanliness.

Reply:
Done